src/views/forms/FormSubWalletCreation/FormSubWalletCreationTs.ts

```diff
@@ -306,7 +334,100 @@ export class FormSubWalletCreationTs extends Vue {
       this.networkType,
       childWalletName,
     )
-
     return wallet
+    }   
+  }
+
+  async importSubAccountFromLedger(childWalletName: string) {
+    const subWalletName = childWalletName
+    const accountPath = this.currentWallet.values.get("path")
+    const currentAccountIndex = accountPath.substring(accountPath.length-2,accountPath.length-1)
+    const numAccount = this.knownPaths.length
+    var accountIndex
+    if(numAccount<= Number(currentAccountIndex) ){
+      accountIndex = numAccount+Number(currentAccountIndex)
+    } else {
+      accountIndex = numAccount+1
+    }
```

Comment:
same note, please use `DerivationPathService`

Reply:
when I use derivation service, there is a problem with the path, because the path of Ledger wallet is: `m/44'/4343'/${networkType}'/0'/${accountIndex}'`, but the path of seed wallet is different: /^m\/44'\/4343'\/[0-9]+'\/[0-9]+'\/[0-9]+'/. So if I use: 
nextPath = this.paths.getNextAccountPath(this.knownPaths), the next path will wrong so we can't connect to Ledger.